### PR TITLE
Fixing decl UID test.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Thanks to @MarinaGryphon for pointing out the issues with the test.

### CODE REVIEW

**Review by MarinaGryphon — APPROVED** [review]
IDK if this review is even valid since, you know, I was the one to find some of this stuff, but also Loaf's a better coder than I am and I got it mostly-working, so.

**Review by afterthought2 — APPROVED** [review]
Classic error.

File: code/datums/outfits/outfit.dm [hunk 1]
```diff
@@ -21,6 +21,7 @@ var/global/list/outfits_decls_by_type_
 
 /decl/hierarchy/outfit
 	name = "Naked"
+	abstract_type = /decl/hierarchy/outfit
```
Comment (PsiOmegaDelta) [inline code/datums/outfits/outfit.dm:24]: This outfit is valid for selection (or at least originally was designed to be a valid choice)

Comment (MistakeNot4892) [reply code/datums/outfits/outfit.dm:24]: It throws a very strange runtime comparing `Naked` against 0.

**Pull request merged**